modules/gapi/include/opencv2/gapi/stereo.hpp

```diff
@@ -47,13 +65,12 @@ G_TYPED_KERNEL(GStereo, <GMat(GMat, GMat, const StereoOutputFormat)>, "org.openc
 
 } // namespace calib3d
 
-/** @brief Extract disparity/depth information depending on passed StereoOutputFormat argument.
-The function extracts disparity/depth information depending on passed StereoOutputFormat argument from
-given stereo-pair.
+/** @brief Computes disparity/depth map for the specified stereo-pair.
+The function compute disparity or depth map depending on passed StereoOutputFormat argument.
 
-@param left left 8-bit unsigned 1-channel image of @ref CV_8UC1 type
-@param right right 8-bit unsigned 1-channel image of @ref CV_8UC1 type
-@param of enum to specify output kind: depth or disparity and corresponding type
+@param left 8-bit single-channel image from left camera.
```

Comment:
What about another sources? Can it be a two videos (left and right)? Clarify it

Reply:
`camera` -> `source`

Reply:
Function description can contain information about possible sources (camera, file, ...).

Reply:
stereo takes two GMat`s as input. So it could be any type of sources, I think...